MiniScopeControlDlg.h

```diff
@@ -197,6 +197,6 @@ class CMiniScopeControlDlg : public CDialogEx
 	afx_msg void OnEnKillfocusMinfluordisplay();
 	afx_msg void OnEnKillfocusMaxfluordisplay();
 	afx_msg void OnCbnCloseupCombo1();
-	int mMSFPS;
+	LONGLONG mMSFPS;
```

Comment:
Curious why mMSFPS was changed to LONGLONG instead of int?

Reply:
I believe I changed this because they got compared with mMSCurrentFPS in the program and somehow I believe the casting of types messed things up, but that's probably wrong and this is not necessary. I can remove this change.